person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative './nameable'
 
 class Person < NameAble
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permision: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permision
+    @rentals = []
     super()
   end
 
```

Comment:
- [ ]   A method for adding rentals should be present.
  ```ruby
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    #*Code here...*
  end
```

Reply:
Okay, what a great view! Thanks a lot. I will walk through it